inst/WORDLIST

```diff
@@ -1,3 +1,4 @@
+ARMCD
```

Comment:
maybe set in backticks to avoid the need for this?

Reply:
it is in the matrix part so adding backtick is ugly and really not necessary


```math
\begin{bmatrix}
\mu & ARMCD & RACE & RACE & ARMCD:RACE & ARMCD:RACE\\
0 & 0 & 1 & 0 & 0 & 0\\ 
0 & 0 & 0 & 1 & 0 & 0\\ 
\end{bmatrix}
```

```math
\begin{bmatrix}
\mu & `ARMCD` & `RACE` & `RACE` & `ARMCD`:`RACE` & `ARMCD`:`RACE`\\
0 & 0 & 1 & 0 & 0 & 0\\ 
0 & 0 & 0 & 1 & 0 & 0\\ 
\end{bmatrix}
```